devel/docker-machine-driver-qemu/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        machine-drivers docker-machine-driver-qemu 0324171328f7bdf2f1e7803dbdc3b18db457c786
+version             20220617
+revision            0
+categories          devel
+platforms           darwin
```

Comment:
The default platform is now `darwin`, so this declaration can be removed.

Reply:
> The default platform is now `darwin`, so this declaration can be removed.

Thanks! I've removed that.